<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_by_amount_wndw</class>
 <widget class="QDialog" name="search_by_amount_wndw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:1, 
        stop:0 rgba(139,0,255, 20), 
        stop:1 rgba(115,102,189, 40));
</string>
  </property>
  <widget class="QFrame" name="frm_table_search_by_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(254,254,34, 5)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QTableView" name="tv_search_by_amount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(254,254,34, 10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frm_search_by_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(254,254,34, 5)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_search_by_amount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
border: none;</string>
    </property>
    <property name="text">
     <string>Поиск по кол-ву на складе</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_enter_amount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
border:none;</string>
    </property>
    <property name="text">
     <string>Введите кол-во</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_enter_amount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_show">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
backgroung-color: rgba(254,254,34, 30);
border: 1px solid rgba(254,254,34, 20);
border-radius: 7px;
}

QPushButton:hover {
background-color: rgba(254,254,34, 40);
}

QPushButton:pressed {
background-color: rgba(254,254,34, 70);
}</string>
    </property>
    <property name="text">
     <string>Вывести</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
